pkg/apis/logging/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,21 +1,5 @@
 // +build !ignore_autogenerated
 
-/*
-Copyright The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
```

Comment:
what happened here?  no code generated for the new `ManagementState` stuff?

Reply:
seems like there isn't anything new to generate since its just a string, i'll revert this file to keep the license info...